engine/src/main/scala/cromwell/jobstore/JobStoreActor.scala

```diff
@@ -18,6 +18,7 @@ import scala.language.postfixOps
   */
 class JobStoreActor(jobStore: JobStore, dbBatchSize: Int, dbFlushRate: FiniteDuration) extends Actor with ActorLogging with GracefulShutdownHelper {
   import JobStoreActor._
+  log.info(s"JobStoreActor configured to write with batch size {} and flush rate {}.", dbBatchSize, dbFlushRate)
```

Comment:
hmm should probably move this and the one below to the actual write actors... ☹️